### ISSUE TITLE
redis pubsub publishes same message to multiple channels

### ISSUE THREAD

**Issue by gittyup2018** [slact/nchan#575]
when subscribed to multiple channels and using redis and websocket pubsub connection , the message gets published to every channel the user is subscribed to, is that  normal behavior?  if so, is there a way to make it to publish to only 1 channel?

**Comment by slact** [comment]
Yes, that's normal, and yes, you can publish to different channels than subscribed to:
https://nchan.io/#pubsub-endpoint

```
  location = /pubsub {
    nchan_pubsub;
    nchan_publisher_channel_id foo;
    nchan_subscriber_channel_id bar;
  }
```

**Comment by gittyup2018** [comment]
Thanks for pointing that out! I wasn't aware it could be so configurable, I guessed I missed that part in the documentation

One more thing I feel dumb asking but is it possible to use upstream request or publisher_channel_id to NOT use redis somehow? I am forwarding data to an application
like in your example so sending it out through [remote] redis is just a waste of bandwidth, latency, etc. 

So I was thinking of using the 204 status to discard the message but actually publish to another endpoint that doesn't' use redis, but not sure if that's possible using one websocket connection? or is there another way?
